app/src/main/java/com/android/unio/model/event/EventViewModel.kt

```diff
@@ -212,25 +212,32 @@ constructor(
    * @param onSuccess A callback that is called when the event is successfully updated.
    * @param onFailure A callback that is called when an error occurs while updating the event.
    */
-  fun updateEventWithoutImage(event: Event, onSuccess: () -> Unit, onFailure: (Exception) -> Unit) {
+  fun updateEventWithoutImage(
+      event: Event,
+      onSuccess: () -> Unit,
+      onFailure: (Exception) -> Unit,
+      updateAssociation: Boolean = true
+  ) {
     repository.addEvent(event, onSuccess, onFailure)
 
-    event.organisers.requestAll(
-        {
-          event.organisers.list.value.forEach {
-            if (it.events.contains(event.uid)) it.events.remove(event.uid)
-            it.events.add(event.uid)
-            associationRepository.saveAssociation(
-                isNewAssociation = false,
-                it,
-                {},
-                { e ->
-                  Log.e("EventViewModel", "An error occurred while loading associations: $e")
-                })
-            it.events.requestAll()
-          }
-        },
-        lazy = true)
+    if (updateAssociation) {
```

Comment:
Very good idea to put the boolean, it allows you not to update all the association as you don't need to here